src/java/org/apache/cassandra/db/marshal/EmptyType.java

```diff
@@ -46,13 +48,12 @@ public class EmptyType extends AbstractType<Void>
     private enum NonEmptyWriteBehavior { FAIL, LOG_DATA_LOSS, SILENT_DATA_LOSS }
 
     private static final Logger logger = LoggerFactory.getLogger(EmptyType.class);
-    private static final String KEY_EMPTYTYPE_NONEMPTY_BEHAVIOR = "cassandra.serialization.emptytype.nonempty_behavior";
     private static final NoSpamLogger NON_EMPTY_WRITE_LOGGER = NoSpamLogger.getLogger(logger, 1, TimeUnit.MINUTES);
     private static final NonEmptyWriteBehavior NON_EMPTY_WRITE_BEHAVIOR = parseNonEmptyWriteBehavior();
 
     private static NonEmptyWriteBehavior parseNonEmptyWriteBehavior()
     {
-        String value = System.getProperty(KEY_EMPTYTYPE_NONEMPTY_BEHAVIOR);
+        String value = SERIALIZATION_EMPTYTYPE_NONEMPTY_BEHAVIOR.getString();
```

Comment:
I would call this like `SERIALIZATION_EMPTY_TYPE_NONEMPTY_BEHAVIOR`

`EMPTYTYPE` -> `EMPTY_TYPE`

Reply:
Fixed.